docs/Examples.md

```diff
@@ -229,19 +215,24 @@ spec:
 EOF
 ```
 
-In the above environment, cluster master controls the [applications loaded](#installing-splunk-apps) to all
-the parts of the indexer cluster, and the indexer services that it creates select the indexers
-deployed by all the IndexerCluster parts, while the indexer services created by indexer cluster only select the indexers that it manages.
+The cluster master controls the [Splunk Apps](#installing-splunk-apps) distributed to all peers in the indexer cluster.  The indexer services that it creates select the indexers deployed by all the IndexerCluster parts, while the indexer services created by indexer cluster only select the indexers that it manages.
 
-This can also allow to better control
-the upgrade cycle to respect the recommended order: cluster master, then search heads,
-then indexers, by defining and updating the docker image used by each IndexerCluster part.
+This allows the Splunk Operator to control the upgrade cycle and use the recommended order: cluster master, search heads, and indexers, by defining and updating the docker image used by each IndexerCluster part.
 
-This solution can also be used to build a [multisite cluster](MultisiteExamples.md)
-by defining a different zone affinity and site in each child IndexerCluster resource.
+This example can be extended to build a [multisite cluster](MultisiteExamples.md) by defining a different zone affinity and site in each child IndexerCluster resource.
 
 The passwords for the instance are generated automatically. To review the passwords, please refer to the [Reading global kubernetes secret object](#reading-global-kubernetes-secret-object) instructions.
 
+
+### Monitoring Console
+The Monitoring Console provides detailed topology and performance information about your Splunk Enterprise deployment. 
+
+Use the `monitoringConsoleRef` paramter to define the name of the MC prior to the pod creation. Or, use `monitoringConsoleRef` to reference a MC pod running in the namespace. The MC pod will periodically check for the existence of new or existing pods in the namespace, and automatically configure the connection used by the MC.
+
+** Add code block example for ADDING MC after a cluster is running
```

Comment:
@kashok-splunk These are the next 3 requests:
** Add code block example for ADDING MC after a cluster is running
** Update CM and Cluster examples above to define the `monitoringConsoleRef`
** Update SHC example below to define the `monitoringConsoleRef`

Let me know if you'd prefer a meeting to discuss these. Thanks!

Reply:
Update SHC: 
`k patch shc test --type=json -p '[{"op":"add", "path":"/spec/monitoringConsoleRef/name", "value":example}]'`
Where `test` is name of SHC and `example ` is name of monitoring console

Update Cluster Master 
`k patch cm-idxc test --type=json -p '[{"op":"add", "path":"/spec/monitoringConsoleRef/name", "value":example}]'`
Where `test` is name of Cluster Master and `example ` is name of monitoring console